Neos.Neos/Classes/Aspects/FusionCachingAspect.php

```diff
@@ -46,4 +56,17 @@ public function cacheGetMergedFusionObjectTree(JoinPointInterface $joinPoint)
 
         return $fusionObjectTree;
     }
+
+    /**
+     * Get a site for the given site node.
+     *
+     * See same method in the FusionService
+     *
+     * @param TraversableNodeInterface $siteNode
+     * @return Site
+     */
+    protected function getSiteForSiteNode(TraversableNodeInterface $siteNode)
```

Comment:
What's the advantage of this method and why don't we just inline the call above?

Reply:
Clearer intention?! Same method name as in FusionService…

Reply:
Exactly, just wanted to point out that this is an exact copy of FusionService behavior and should stay in sync.

Reply:
Fair enough, I added a `@see` comment to link it closer to the corresponding method now